testing/endtoend/types/types.go

```diff
@@ -94,6 +94,13 @@ type E2EConfig struct {
 
 func GenesisFork() int {
 	cfg := params.BeaconConfig()
+	// Check from highest fork to lowest to find the genesis fork.
```

Comment:
1. Can we already add Fulu here?
2. What's the difference between this function and the `genesisFork()` in `testing/endtoend/evaluators/fork.go`? Can't we just use this one, which is already exported?